packages/react/src/components/MapCard/storyFiles/Optionsfield.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  changeState: PropTypes.func.isRequired,
+  property: PropTypes.string.isRequired,
+  options: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string,
+      property: PropTypes.string,
+    })
+  ).isRequired,
+};
+
+const Optionsfield = ({ changeState, property, options }) => {
+  const renderOptions = (option, i) => {
+    return (
+      <label htmlFor={option.name} key={i} className="toggle-container">
```

Comment:
Should consider using something unique to the value rather than the index

Reply:
fixed